### PR TITLE
Add patient listing to landing page to select patient to launch ASCVD app with. Fixes #23

### LINKED ISSUE #23: Add multiple test patients to the landing page
Our landing page current redirects to a single hardcoded patient launch. Instead, it would be better if we had a list of test patients, each being linked to a launch for that particular patient id.

### PR DESCRIPTION
**Pull request by zplata**
Currently, the way we display the app public-facing is with a URL to `http://cerner.github.io/cardiac-risk-app/landing.html` which automatically redirects to a URL that launches the app with a given FHIR server and a hardcoded patient ID. 

We want to be able to show a simple list of patients on the landing page and describe each patient at a high level overview of what their medical history looks like, along with gender and age. With this addition, someone trying to simply run the application can have a multitude of patients to choose from for context. Each patient will have a redirect URL to launch the app with a given FHIR server the patient is hosted in, as well as the patient ID of that particular patient.

@kpshek 
@mjhenkes 
@kolkheang
@koushic88
@shriniketsarkar

### CODE REVIEW

**Comment by shriniketsarkar** [comment]
👍

**Comment by koushic88** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ For our model of an improved Reynolds Risk Score reporting SMART App, we took an
 App Design:
 ![Sample App Design Screen Shot](CardiacRiskApp.png)
 
+# Try the App #
+
+You can choose from a list of patients and [run the app here](http://cerner.github.io/cardiac-risk-app/patients.html).
```
Comment (mjhenkes) [inline README.md]: I could be missing something, but how does patients.html redirect to landing.html?

Comment (zplata) [inline README.md]: I originally had a file called patients.html with these patients but last minute decided to just put it all in landing.html. Thanks for that good catch @mjhenkes ! Fixed

**Comment by mjhenkes** [comment]
+1

**Pull request closed without merging**